editor/import/audio_stream_import_settings.cpp

```diff
@@ -437,7 +437,7 @@ void AudioStreamImportSettingsDialog::edit(const String &p_path, const String &p
 			beats_edit->set_value(beats);
 			beats_enabled->set_pressed(beats > 0);
 			loop->set_pressed(config_file->get_value("params", "loop", false));
-			loop_offset->set_value(config_file->get_value("params", "loop_offset", 0));
+			loop_offset->set_value(double(config_file->get_value("params", "loop_offset", 0)));
```

Comment:
Similarly, `Range::set_value()` takes a `double`, so this cast doesn't do anything that didn't happen before.

Reply:
See above

Reply:
_(copied from previous conversation that was marked resolved)_

Here it is reading from a VARIANT::FLOAT afiak (I know that it's a double under the hood… but there should still be in implicit conversion going on that we can make an explicit cast)

Reply:
Indeed